src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsFormWithGuiFilters.tsx

```diff
@@ -318,15 +318,16 @@ class RuleConditionsFormWithGuiFilters extends React.PureComponent<Props, State>
             </FormRow>
             {this.props.thresholdChart}
           </List>
-        </PanelBody>
-      </StyledPanel>
+        </StyledPanelBody>
+      </Panel>
     );
   }
 }
 
-const StyledPanel = styled(Panel)`
-  /* Sticky graph panel cannot have margin-bottom */
-  padding: ${space(1)};
+const StyledPanelBody = styled(PanelBody)`
+  h4 {
```

Comment:
had to override a style here because all `h4`'s in the PanelBody have `margin: 20px` by default, which caused a lot of whitespace to be added to the bottom of the `<SectionHeading>` for total counts